robotpy_ext/control/toggle.py

```diff
@@ -0,0 +1,107 @@
+from _functools import partial
+
+
+class Toggle:
+    """Utility class for button toggle
+
+        Usage:
+
+        foo = Toggle(joystick, 3)
+
+        if foo:
+            toggleFunction()
+
+        if foo.on:
+            onToggle()
+
+        if foo.off:
+            offToggle()
+    """
+    class _SteadyDebounce:
+        '''
+            Similar to ButtonDebouncer, but the output stays steady for
+            the given periodic_filter. E.g, if you set the period to 2
+            and press the button, the value will return true for 2 seconds.
+
+            Steady debounce will return true for the given period, allowing it to be
+            used with Toggle
+        '''
+
+        def __init__(self, joystick, button, period=0.5):
+            from wpilib.timer import Timer
```

Comment:
Not sure what the general consensus is with importing when you need to vs just doing it anyway. Should it be at the top?